tests/test_validators.py

```diff
@@ -1,23 +1,34 @@
 import pytest
-from name.models import validate_merged_with, Name
+from name.validators import validate_merged_with
```

Comment:
These imports should be grouped by type and separated by a blank line according to PEP-8 standards